modules/mono/editor/GodotTools/GodotTools.IdeMessaging/Utils/NotifyAwaiter.cs

```diff
@@ -25,7 +28,7 @@ public void OnCompleted(Action continuation)
             this.continuation = continuation;
         }
 
-        public void SetResult(T result)
+        public void SetResult(T? result)
```

Comment:
I don't think this should be nullable.

```suggestion
        public void SetResult(T result)
```